### PR TITLE
Calyx: Initial commit.

### PR DESCRIPTION
**Pull request by alaczek**
This is a new single-page theme based on this pattern: https://dotcompatterns.wordpress.com/2021/05/20/coming-soon-9/

### CODE REVIEW

File: calyx/style.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+/*
+ * Button hover styles.
+ * Necessary until the following issue is resolved in Gutenberg:
+ * https://github.com/WordPress/gutenberg/issues/27075
+ */
+
+.wp-block-search__button:hover,
+.wp-block-file .wp-block-file__button:hover,
+.wp-block-button__link:hover {
+	background-color: var(--wp--preset--color--secondary);
```
Comment (scruffian) [inline calyx/style.css]: This should be possible in theme.json,

File: calyx/style.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+ * modified too heavily by themes or plugins that build on
+ * Twenty Twenty-Two. These are meant to be a precursor to
+ * a global solution provided by the Block Editor.
+ *
+ * Relevant issues:
+ * https://github.com/WordPress/gutenberg/issues/35607
+ * https://github.com/WordPress/gutenberg/issues/35884
+ */
+
+.wp-site-blocks,
```
Comment (scruffian) [inline calyx/style.css]: These rules aren't needed anymore

**Review by scruffian — COMMENTED** [review]
I think you can remove the style.css and therefore functions.php as well.

File: calyx/style.css (hunk 1 above)
Comment (alaczek) [reply calyx/style.css]: Good call, I removed it.

File: calyx/style.css (hunk 2 above)
Comment (alaczek) [reply calyx/style.css]: Good call! Removed!

**Comment by alaczek** [comment]
I'm going to merge this one so that I can start setting it up on .com.

**Pull request merged**